extension/docs/DEV.md

```diff
@@ -45,18 +45,16 @@ cp .env.example .env.production
 git submodule update --init --recursive
 ```
 
-2. Add the model files to be included in the build according to the instructions [here](../bergamot-translator/README.md).
+2. Acquire build artifacts bergamot-translator, according to the instructions [here](../bergamot-translator/README.md).
 
-3. Run the build and import script (in the `extension/` folder in this repo):
+3. Put the bergamot-translator build artifacts in the `../bergamot-translator/build-wasm-docker/wasm/` folder, then run the following import script (from the `extension/` folder in this repo):
```

Comment:
What is hindering this repo from building bergamot-translator as well instead one being required to manually do this step?

Reply:
Currently bergamot-translator suffers from non-reproducible builds. There are multiple issues filed (and some not filed from the internal development) about this, so as soon as that is fixed, the extension will also be able to build bergamot-translator reliably.